rootdir/system/etc/unblock_wakelock.sh

```diff
@@ -1,14 +1,14 @@
 #!/system/bin/sh
 
-#wakelock=sns_periodic_wakelock
+wakelock=sns_periodic_wakelock
```

Comment:
Did I open the PR against the wrong branch=

Reply:
The branch is correct, wondering if you branched off an older state of the repo.

Reply:
Fixed it.